java/activej/pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <version>1.2.4</version>
+            <version>1.2.5</version>
```

Comment:
@waghanza we do not really care about logging dependency version. 
But we need to specify some version, maybe we can do `[1.2, 1.3)` so that dependency bot would not trigger?
I'm not very familiar with `renovate`

Reply:
understood @eduard-vasinskyi. however, is there not any logging `built-in` feature in `activej` ? I mean, do we have to explicitly define `logback` ? is it not included by default ?

Reply:
@waghanza 
ActiveJ uses [SLF4J](http://www.slf4j.org/) which is a logging facade, so that the framework is not directly dependent on any logging library.